docs/UserGuide.md

```diff
@@ -108,6 +108,20 @@ Examples:
 * `add n/John p/92781123 e/john@example.com a/Donald street, block 248, #02-04 g/m`
 * `add n/Charlotte p/81286623 e/charlotte@example.com a/Charity street, block 101, #10-82  g/F`
 
+__Optional Parameter 2: Date of Birth__
+
+Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [d/DOB]`
+`
+
+- Adds date of birth to a person in the contact list. Date formats accepted are: 		dd.MM.yyyy , dd MMM yyyy, M d, yyyy
```

Comment:
Perhaps, can remove the huge whitespace between are: and dd.MM.yyy & use ; to separate the formats since comma is already used in the third format
```suggestion
- Adds date of birth to a person in the contact list. Date formats accepted are: dd.MM.yyyy ; dd MMM yyyy ; M d, yyyy
```

Reply:
thanks for catching this too